fiftyone/core/threed/transformation.py

```diff
@@ -60,4 +118,29 @@
         return np.array([self.x, self.y, self.z, self.w])
 
 
-Vec3UnionType = Union[Vector3, List[float], Tuple[float]]
+Vec3UnionType = Union[Vector3, List[float], Tuple[float], np.array]
+
+
+def normalize_to_vec3(v: Optional[Vec3UnionType]) -> Union[Vector3, None]:
+    if v is None or isinstance(v, Vector3):
+        return v
+
+    try:
+        item_list = validate_list(v, 3)
+
+    except ValueError as e:
+        raise ValueError(
+            "Expected a list / tuple of length 3 or a Vector3"
+        ) from e
+
+    return Vector3(*item_list)
+
```

Comment:
This doesn't work because it would identify non-numerical scalars such as strings as well.
From documentation here: https://numpy.org/doc/stable/reference/generated/numpy.isscalar.html
> If you need a stricter way to identify a numerical scalar, use isinstance(x, numbers.Number), as that returns False for most non-numerical elements such as strings.

So I will use `numbers.Real` instead of `np.number`.